packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -263,3 +345,29 @@ export function evalBlobsAndTrees(snapshot: IOdspSnapshot) {
     }
     return { numTrees, numBlobs, encodedBlobsSize, decodedBlobsSize };
 }
+
+export function isRedeemSharingLinkError(odspResolvedUrl: IOdspResolvedUrl, error: any) {
+    const errorType = error.errorType;
```

Comment:
there should be a check that typeof error == "object" &&  error !== null here (and bail out if it's not).
Comment says you add it, but it's not here :)